ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py

```diff
@@ -2051,7 +2051,6 @@ def test_paddle_isclose(
     )
 
 
-
```

Comment:
Please try not to edit other tests.

Reply:
hi, was trying to resolve the merge conflict 😥

Reply:
Ah if it's `black` kicking in with the autoformatting then it's fine, my bad again.